local-dev/templates/rawls.conf.ctmpl

```diff
@@ -165,6 +165,7 @@ avroUpsertMonitor {
   pubSubProject = "broad-dsde-dev"
 
   server = "https://terra-importservice-dev.appspot.com"
+  cwds = "https://cwds.dsde-dev.broadinstitute.org"
```

Comment:
similar change to https://github.com/broadinstitute/terra-helmfile/pull/5233, but adds support for local dev